src/uct/ib/ud/base/ud_iface.c

```diff
@@ -389,10 +389,10 @@ static void uct_ud_iface_calc_gid_len(uct_ud_iface_t *iface)
     uint16_t *local_gid_u16 = (uint16_t*)iface->super.gid.raw;
 
     /* Make sure that daddr in IPv4 resides in the last 4 bytes in GRH */
-    UCS_STATIC_ASSERT((UCT_IB_GRH_LEN - (20 + offsetof(struct iphdr, daddr))) == ipv4_len);
+    ucs_assert_always((UCT_IB_GRH_LEN - (20 + offsetof(struct iphdr, daddr))) == ipv4_len);
```

Comment:
pls use #define UCT_UD_IPV4_LEN to use static assert - rather that reverting to runtime assert